api/filters/replacement/replacement.go

```diff
@@ -6,6 +6,7 @@ package replacement
 import (
 	"errors"
 	"fmt"
+	utils2 "sigs.k8s.io/kustomize/kyaml/utils"
```

Comment:
fix imports, and lets call this something more descriptive, maybe `kyaml_utils` instead of `utils2`